src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerOfflineDocumentVerified.java

```diff
@@ -282,4 +285,42 @@ private DivorceDocument mapScannedDocumentToDivorceDocument(final ScannedDocumen
             .documentComment("Reclassified scanned document")
             .build();
     }
+
+    public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details, CaseDetails<CaseData, State> beforeDetails) {
+
+        final CaseData caseData = details.getData();
+
+        if (CO_D84.equals(caseData.getDocuments().getTypeOfDocumentAttached())) {
+            notificationDispatcher.send(app1AppliedForConditionalOrderNotification, caseData, details.getId());
+
+            if (SWITCH_TO_SOLE.equals(caseData.getConditionalOrder().getD84ApplicationType())) {
+
+                log.info(
+                    "CaseworkerOfflineDocumentVerified submitted callback triggering SwitchedToSoleCO event for case id: {}",
+                    details.getId());
+
+                final User user = idamService.retrieveSystemUpdateUserDetails();
+                final String serviceAuth = authTokenGenerator.generate();
+                ccdUpdateService.submitEvent(details, SWITCH_TO_SOLE_CO, user, serviceAuth);
```

Comment:
as discussed if SWITCH_TO_SOLE_CO event fails then we need to have some way of retrying the event. Either we can add Super user access to SWITCH_TO_SOLE_CO or we can provide a retry event for caseworkers